examples/jaeger-skaffold-trace/README.md

```diff
@@ -0,0 +1,37 @@
+### Example: Skaffold Command Tracing with Jaeger
+
```

Comment:
Can you put a note here that this is experimental and may change without notice.  And be sure to place it in integration/examples too

Reply:
Done